hdf5-vol-daos.spec

```diff
@@ -197,6 +193,12 @@ done
 %endif
 
 %changelog
+* Fri Jan 15 2021 Kenneth Cain <kenneth.c.cain@intel.com> - 0.1-5.gfcbdc0b
+- Update hdf5-vol to fcbdc0b and virtual provides to build with libdaos.so.1
+
+* Fri Jan 15 2021 Kenneth Cain <kenneth.c.cain@intel.com> - 0.1-4.g17234b2
+- Update hdf5-vol to g17234b2 and remove unnecessary Requires: lines
+
```

Comment:
These appear to be both in this same PR.  There is no need to have multiple Releases in a single commit.  Could wind up being quite confusing at some point in the future even.  Combine these into a single new Release.

Reply:
Agree with this. It is a byproduct of some work done very recently in which it was thought that daos might need to be brought in sync with the intermediate vol-daos source commit 17234b2. This came in through a merge from that other experimental work.

I can fix this up and make a single new 0.1.4-gfcbdc0b release.